ivy/functional/frontends/numpy/ndarray/ndarray.py

```diff
@@ -723,3 +723,14 @@ def __rshift__(self, value, /):
 
     def __lshift__(self, value, /):
         return ivy.bitwise_left_shift(self.ivy_array, value)
+
+    def var(self, *, axis=None, dtype=None, out=None, keepdims=False, where=True, ddof=0):
```

Comment:
Would you please update these to have the same order as in the original numpy function :https://numpy.org/doc/stable/reference/generated/numpy.var.html